docs/deployment_patterns/how_to_use_great_expectations_in_databricks.md

```diff
@@ -20,8 +20,8 @@ There are several ways to set up Databricks, this guide centers around an AWS de
 
 We will cover a simple configuration to get you up and running quickly, and link to our other guides for more customized configurations. For example:
   - If you want to validate files stored in DBFS select one of the "File" tabs below. You can also [watch our video walkthrough](https://youtu.be/9mIsmyuJzhQ) of these steps.
-    - If you are using a different file store (e.g. s3, GCS, ABS) take a look at our how-to guides in the "Cloud" section of "Connecting to Your Data" for example configurations. 
-  - If you already have a spark dataframe loaded, select one of the "Dataframe" tabs below. 
```

Comment:
This file had some minor formatting changes that were introduced as part of `lint`.